### DISCUSSION TITLE
Welcome to Backend Discussions!

### DISCUSSION THREAD

**Discussion by ChaiShalev18** [Carpool-Buddies/Backend#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saberhelaly** [comment]
I am Saber Hilali, a beginner in programming. I created my first project and I want a simple modification. Can someone help me upload it to the host?
